src/main/java/io/wisetime/connector/jira/JiraConnector.java

```diff
@@ -208,4 +209,18 @@ private String tagUpsertPath() {
   private Optional<String> callerKey() {
     return RuntimeConfig.getString(ConnectorConfigKey.CALLER_KEY);
   }
+
+  /**
+   * @return an array of projects keys used in upserting tags
+   */
+  @VisibleForTesting
+  String[] onlyUpsertTagsForProjectKeys() {
+    return RuntimeConfig
+        .getString(JiraConnectorConfigKey.ONLY_UPSERT_TAGS_FOR_PROJECT_KEYS)
```

Comment:
If this is an optional config,  can we confirm this will throw NPE when called?

Reply:
👍